app/src/main/java/com/example/happymeals/userlogin/RegisterActivity.java

```diff
@@ -1,23 +1,25 @@
-package com.example.happymeals;
+package com.example.happymeals.userlogin;
 
 import android.annotation.SuppressLint;
 import android.content.Intent;
 import android.os.Bundle;
 import android.text.TextUtils;
+import android.util.Log;
 import android.view.View;
 import android.widget.Button;
 import android.widget.EditText;
 import android.widget.TextView;
 
 import androidx.appcompat.app.AppCompatActivity;
 
-import com.example.happymeals.database.FireAuth;
+import com.example.happymeals.R;
+import com.example.happymeals.database.FirebaseAuthenticationHandler;
 
-public class Register extends AppCompatActivity{
-    private Button register;
+public class RegisterActivity extends AppCompatActivity{
+    private Button registerBtn;
     private TextView returnLogin;
-    private EditText user, password;
-    private FireAuth fireAuth;
+    private EditText userData, passData;
```

Comment:
Let's have a talk about the naming of these. Data implies it holds information and is a data type. These are custom view fields in our .xml files. The name of the variables should reflect this.